assignments/todo_list/todo.html

```diff
@@ -5,8 +5,10 @@
     <body>
         <div>
             <input type='text' name='todo_input'>
-            <button type='submit'>Add Todo</button>
-            <div class='todos'></div>
+            <button onclick="submit()" type='submit'>Add Todo</button>
```

Comment:
you didn't used onclick attribute, submit func isn't defined as well.

Reply:
yes actually i used this for testing purpose but using this method testcases are not passed thats why i am not using in script